### PR TITLE
Fix a problem where a player never sent a message, joined a game and was almost unbannable.

### PR DESCRIPTION
**Pull request by Olfi01**
I just forked this to one of my organizations bcs i couldnt find how to delete all those commits, sorry

### CODE REVIEW

File: Werewolf for Telegram/Werewolf Node/Werewolf.cs [hunk 1]
```diff
@@ -284,7 +284,7 @@ private void GameTimer()
 
                     if (secondsElapsed++ == 30 && Players.Any(x => !notifiedPlayers.Contains(x.Id))) //every 30 seconds, tell in group who have joined
                     {
-                        SendWithQueue(GetLocaleString("HaveJoined", Players.Where(x => !notifiedPlayers.Contains(x.Id)).Aggregate("", (cur, p) => cur + p.GetName() + ", ").TrimEnd(',', ' ')));
+                        SendWithQueue(GetLocaleString("HaveJoined", Players.Where(x => !notifiedPlayers.Contains(x.Id)).Aggregate("", (cur, p) => cur + p.GetName() + §" (ID: {p.Id}), ").TrimEnd(',', ' ')));
```
Comment (renyhp) [inline Werewolf for Telegram/Werewolf Node/Werewolf.cs]: Actually, the character to format a string is $, not §...

Comment (renyhp) [inline Werewolf for Telegram/Werewolf Node/Werewolf.cs]: Also, p.Id is not the player's Telegram id... should be p.TeleUser.Id or something similar

**Comment by Olfi01** [comment]
1. Oh yes typo
2. Damn sorry can't fix that rn bcs i haven't got my lappy here 😅

**Comment by Olfi01** [comment]
Now. For real. I even checked with the database file.

File: Werewolf for Telegram/Werewolf Node/Werewolf.cs (hunk 1 above)
Comment (Olfi01) [inline Werewolf for Telegram/Werewolf Node/Werewolf.cs]: Oh that damned mobile version didn't lemme comment on the review sorry

**Pull request merged**